view/frontend/web/js/hooks.js

```diff
@@ -12,10 +12,17 @@
  **/
 
 // NOTE: The algoliaRecentSearches dependency is optional and is only supplied for demonstration of inclusion of the recent searches plugin
-define(['jquery', 'algoliaAnalytics', 'algoliaBundle', 'suggestionsHtml', 'algoliaRecentSearches', 'algoliaCommon'], function (
+define([
+    'jquery',
+    'algoliaAnalytics',
+    'algoliaQuerySuggestionsPluginLib',
+    'suggestionsHtml',
+    'algoliaRecentSearches',
```

Comment:
`algoliaRecentSearches` was previsouly packaged with the `CustomAlgolia` extension as a demo but it's actually in the bundle we are unpacking. I think it would be good if we could sync this module with that upcoming change and include a reference to the new location. We should probably adopt the new naming convention too I think?